portfolio/src/main/webapp/script.js

```diff
@@ -31,23 +31,191 @@ function addRandomFunFact() {
         unusedFunFacts[randomIndex] = firstFunFact;
         unusedFunFacts.shift();
 
-        const funFactRowText = '<tr><td>' + funfact + '</td></tr>';
-        tableText += '<tr><td>' + funfact + '</td></tr>';
-        countTableEntries++;
+        factTableText += '<tr><td>' + funfact + '</td></tr>';
         document.getElementById('funfact-message').innerHTML = "";
         
-        funFactTable.innerHTML = tableText + '</table>';
+        funFactTable.innerHTML = factTableText + '</table>';
 
   } else {
       document.getElementById('funfact-message').innerHTML = "Congrats - you've gone through all of the fun facts!";
   }
 
 }
 
-//outside of function because we want to add to it each time addRandomFunFact is called
-var tableText = `<table style="width:100%"> 
+/**
+ * Fetches the comments from the server and adds it to the DOM.
+ Based on week-3-server/random-quotes/src/webapp/script.js from the Week-3-Server tutorial
+ */
+function getComments(myLocation) {
+
+  // First, reset by updating num comments, which also hides comments
+  refreshNumComments(myLocation);
+  
+  // Toggle between hide and display
+  if (shouldDisplay) {
```

Comment:
I think i understand what this boolean is now - maybe name it

showingDisplayCommentButton